### PR TITLE
Refactored tests

### PR DESCRIPTION
**Pull request by chris-ianson**

### CODE REVIEW

**Comment by chris-ianson** [comment]
@RonanOCarroll These look great, thanks.

File: test/uk/gov/hmrc/brm/controllers/BirthEventsControllerSpec.scala [hunk 1]
```diff
@@ -116,27 +116,51 @@ class BirthEventsControllerSpec
         val result = await(MockController.post().apply(request))
         checkResponse(result,BAD_REQUEST, MockErrorResponses.INVALID_FIRSTNAME.json)
       }
+
+      "return response code 400 if request contains a single space in firstName" in {
+        val request = postRequest(firstNameWithASingleSpace)
```
Comment (adamconder) [inline test/uk/gov/hmrc/brm/controllers/BirthEventsControllerSpec.scala:121]: @chris-ianson good find!

File: test/uk/gov/hmrc/brm/models/PayloadSpec.scala [hunk 2]
```diff
@@ -204,97 +118,48 @@ class PayloadSpec extends UnitSpec with WithFakeApplication {
...
-            | "birthReferenceNumber": "123456789",
-            | "firstName" : "John\n",
-            | "lastName" : "Smith",
-            | "dateOfBirth" : "1997-01-13",
-            | "whereBirthRegistered" : "england"
-            |}
-          """.stripMargin)
-
-        jsonObject.validate[Payload].isError shouldBe true
+        val payload = Payload(None, "John\n", None, "Test", LocalDate.now, BirthRegisterCountry.ENGLAND)
```
Comment (adamconder) [inline test/uk/gov/hmrc/brm/models/PayloadSpec.scala:131]: @chris-ianson What was the decision to move to using the case class in the spec?

Comment (chris-ianson) [reply test/uk/gov/hmrc/brm/models/PayloadSpec.scala:131]: @adamconder We were reordering the tests and thought some of the tests were easier to read across two lines.

**Pull request merged**